core/schema/modulesource.go

```diff
@@ -240,7 +240,7 @@ type buildkitClient interface {
 // - if nothing worked, fallback as local ref, as before
 func parseRefString(ctx context.Context, bk buildkitClient, refString string) parsedRefString {
 	var parsed parsedRefString
-	parsed.modPath, parsed.modVersion, parsed.hasVersion = strings.Cut(refString, "@")
+	parsed.modPath = refString
```

Comment:
nit: the check right below here for `!parsed.hasVersion` will always be `true` now, so it's a bit redundant + confusing. I think with the new logic we can just remove that check now?

Reply:
You're right, corrected 🙏